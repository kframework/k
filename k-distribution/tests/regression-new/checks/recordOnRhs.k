module RECORDONRHS
  imports INT

syntax Foo ::= foo(Int)

rule 0 => foo(...)

endmodule
