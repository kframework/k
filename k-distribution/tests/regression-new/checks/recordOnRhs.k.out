[Error] Compiler: Found variable _Gen0 on right hand side of rule, not bound on left hand side. Did you mean "?_Gen0"?
	Source(recordOnRhs.k)
	Location(6,11,6,19)
[Error] Compiler: Had 1 structural errors.
